src/app/@ansyn/core/utils/extent-calculator.ts

```diff
@@ -1,24 +1,24 @@
 import { CaseMapExtentPolygon } from '@ansyn/core/models/case-map-position.model';
 import { toRadians } from './math';
-import { cloneDeep } from 'lodash';
 import * as turfCenter from '@turf/center';
 import { CaseMapExtent } from '@ansyn/core';
+import * as turf from '@turf/turf';
 
 export class ExtentCalculator {
 
 	static polygonToExtent(extentPolygon: CaseMapExtentPolygon): CaseMapExtent {
 		return <CaseMapExtent> [...extentPolygon.coordinates[0][0], ...extentPolygon.coordinates[0][2]];
 	}
 
-	static extentToPolygon(extent: CaseMapExtent): CaseMapExtentPolygon {
+	static extentToPolygon(extent: CaseMapExtent): GeoJSON.Feature<GeoJSON.Polygon> {
 		const minX = extent[0];
 		const minY = extent[1];
 		const maxX = extent[2];
 		const maxY = extent[3];
 
 		const coordinates = [[minX, minY], [minX, maxY], [maxX, maxY], [maxX, minY], [minX, minY]];
 
-		return { type: 'Polygon', coordinates: [coordinates] };
+		return turf.polygon([coordinates]);
```

Comment:
it's a feature! should be geometry

Reply:
I'm expecting a feature, look at the usages of this function.